app/src/pages/inside/launchesPage/launchesPage.jsx

```diff
@@ -207,6 +220,18 @@ export class LaunchesPage extends Component {
     this.props.unselectAllLaunchesAction();
   }
 
+  getNoItemMessage = () => (
```

Comment:
Better to rename this method to `renderNoItemMessage` because it return some markup but not a value or something.